src/diffusers/loaders/single_file_utils.py

```diff
@@ -1381,65 +1402,86 @@ def create_scheduler_from_ldm(
     prediction_type=None,
     scheduler_type="ddim",
     model_type=None,
+    is_playground_model=False,
 ):
-    scheduler_config = get_default_scheduler_config()
-    model_type = infer_model_type(original_config, model_type=model_type)
```

Comment:
Can't we use `model_type` set to `Playground` instead of an `is_playground_model`?